force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -19,10 +19,21 @@
 trigger ContactTrigger on Contact(before insert) {
```

Comment:
Make sure you include all the contexts you are using (after insert, after update)